src/app/Components/ContextMenu/ContextMenuArtwork.tsx

```diff
@@ -229,7 +216,7 @@ export const ContextMenuArtwork: React.FC<ContextMenuArtworkProps> = ({
           underlayColor={color("white100")}
           activeOpacity={0.8}
           onLongPress={handleAndroidLongPress}
-          delayLongPress={1200}
+          delayLongPress={400}
```

Comment:
Was there a reason to use a different `delayLongPress` value in this component and in `ArtworkGridItem` and `ArtworkRailCard` components? If not, let's keep it consistent 400 🚀 
cc @olerichter00

Reply:
Yes, the `1200` is to avoid the context menu from opening on a (long) normal press on Android.

```suggestion
          delayLongPress={1200} // To avoid the context menu from opening on a (long) normal press on Android.
```

Reply:
👌 Added the 1200 delay back, had to adjust the tracking due to this